plugins/quickinspector/quickinspector.cpp

```diff
@@ -228,6 +229,17 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
   return QStringLiteral("Unknown: %1").arg(wrapMode);
 }
 
+static bool qGoodCandidateItem(QQuickItem *item)
```

Comment:
Just tried to stay in the line of the previous members which also prefix with q.
But it's absolutely not linked to Qt itself.